src/pages/Detail/Detail.js

```diff
@@ -1,8 +1,80 @@
 import React, { Component } from 'react';
+import DetailGallery from './DetailGallery/DetailGallery';
+import DetailInfo from './DetailInfo/DetailInfo';
+import Nav from '../../components/Nav/Nav';
+import TCLogin from '../../components/TestComp/TCLogin';
+import './Detail.scss';
 
 export class Detail extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      detailData: [],
+      loginModal: false,
+    };
+  }
+
+  componentDidMount() {
+    this.detailData();
+  }
+
+  detailData() {
+    fetch('/data/ysLim/detailData.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          detailData: data[0],
+        });
+      });
+  }
+
+  toggleLogin = () => {
+    const { loginModal } = this.state;
+
+    this.setState({
+      loginModal: !loginModal,
+    });
+  };
+
   render() {
-    return <div>'Detail!'</div>;
+    const { loginModal, detailData } = this.state;
+    const {
+      id,
+      serial,
+      title,
+      sub_title,
+      price,
+      colors,
+      eco_friendly,
+      quantity,
+      description_head,
+      description,
+      images,
+    } = detailData;
+
+    return (
+      <>
+        <Nav toggle={this.toggleLogin} />
+        {loginModal ? <TCLogin toggle={this.toggleLogin} /> : null}
```

Comment:
```suggestion
        {loginModal && <TCLogin toggle={this.toggleLogin} />}
```
굳이 삼항연산자 사용 안하셔도 됩니다!